services/ui-src/src/utils/validation/schemas.test.ts

```diff
@@ -14,7 +14,7 @@ describe("Schemas", () => {
     "N/A",
     "Data not available",
   ];
-  const badNumberTestCases = ["abc", "N", "", "123:123", "!@#!@%"];
+  const badNumberTestCases = ["abc", "N", "", "!@#!@%"];
```

Comment:
Question: Do we need the `""` test case here or can we remove it altogether?

Reply:
still a bad number 🤷 we don't want to allow empty string